charts/kafka/charts/zookeeper/values.yaml

```diff
@@ -122,7 +122,7 @@ persistence:
   ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
   ##   GKE, AWS & OpenStack)
   ##
-  # storageClass: "-"
+  storageClass: "kafka-persistence"
```

Comment:
update the header. Need to add the crown copyright, apache 2 licence header and change the 'Copied from: ...' to 'Originally based on ...'